Makefile

```diff
@@ -148,6 +145,9 @@ thrift_gen: $(BIN)/thrift
 	PATH=$(BIN):$$PATH $(BUILD)/thrift-gen --generateThrift --inputFile examples/keyvalue/keyvalue.thrift --outputDir examples/keyvalue/gen-go
 	PATH=$(BIN):$$PATH $(BUILD)/thrift-gen --generateThrift --inputFile examples/thrift/example.thrift --outputDir examples/thrift/gen-go
 	PATH=$(BIN):$$PATH $(BUILD)/thrift-gen --generateThrift --inputFile hyperbahn/hyperbahn.thrift --outputDir hyperbahn/gen-go
+	PATH=$(BIN):$$PATH $(BUILD)/thrift-gen --generateThrift --inputFile thrift/meta.thrift --outputDir thrift/gen-go
+	rm thrift/gen-go/meta/tchan-meta.go # circular dependency, as we just want to generate thrift files here
```

Comment:
It looks like previously, they would manually generate the tchan thrift files with thrift-gen and check them in. This re-uses the compiled thrift-gen and removes the tchannel-go parts. So this is kind of a hack to automate the use of thrift-gen.